com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added a slider that controls how much the volumetric clouds erosion value affects the ambient occlusion term.
 - Added three animation curves to control the density, erosion, and ambient occlusion in the custom submode of the simple controls.
 - Added support for the camera bridge in the graphics compositor
+- Added a dialog box on import to add a diffusion profile to the global settings.
```

Comment:
Tweaked for clarity: 

Added a dialog box when you import a Material that has a diffusion profile to add the diffusion profile to global settings.